### ISSUE TITLE
changing `dbVersion` in config.json is causing pg_bouncer  or auth errors   

### ISSUE THREAD

**Issue by osaxma** [supabase/cli#101]
# Bug report

## Describe the bug

After a fresh `supabase init`, I change the default `dbVersion` from `130003` to `140001` to match the remote version. When I do so, this is what I see in the studio under database, editor  and auth  tabs (same error appears when running a query):

<img width="409" alt="Screen Shot 2021-12-08 at 5 33 15 PM" src="https://user-images.githubusercontent.com/46427323/145231325-988b6ac0-18cc-4b20-874b-39be606ec054.png">

## Steps to Reproduce:
1-  init a project
```
[~] mkdir fresh_supabase && cd fresh_supabase
[~/fresh_supabase] git init . && supabase init 
```
2- In `config.json`, Change `"dbVersion" :"130003"` to `"dbVersion": "140001"`
3- run `supabase start`. 
4- The screenshot above should appear under the database or auth tabs. 

## System information

- OS: macOS BigSur 11.6.1 (20G224)
- supabase cli version: `0.15.5%`

## Logs with errors 
Here are the logs from different containers (some of them stopped running and they're noted below):

<details><summary> Click to Expand: Postgres Container logs (running)</summary>

```
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.UTF-8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data ... ok
creating subdirectories ... ok
selecting dynamic shared memory implementation ... posix
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting default time zone ... Etc/UTC
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok


Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data -l logfile start

waiting for server to start....2021-12-08 15:16:23.601 UTC [72] LOG:  starting PostgreSQL 14.1 (Debian 14.1-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2021-12-08 15:16:23.603 UTC [72] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2021-12-08 15:16:23.608 UTC [75] LOG:  database system was shut down at 2021-12-08 15:16:23 UTC
2021-12-08 15:16:23.614 UTC [72] LOG:  database system is ready to accept connections
 done
server started

/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

2021-12-08 15:16:23.720 UTC [72] LOG:  received fast shutdown request
waiting for server to shut down....2021-12-08 15:16:23.723 UTC [72] LOG:  aborting any active transactions
2021-12-08 15:16:23.725 UTC [72] LOG:  background worker "logical replication launcher" (PID 81) exited with exit code 1
2021-12-08 15:16:23.725 UTC [76] LOG:  shutting down
2021-12-08 15:16:23.740 UTC [72] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

initdb: warning: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
2021-12-08 15:16:23.841 UTC [1] LOG:  starting PostgreSQL 14.1 (Debian 14.1-1.pgdg110+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 10.2.1-6) 10.2.1 20210110, 64-bit
2021-12-08 15:16:23.841 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2021-12-08 15:16:23.841 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2021-12-08 15:16:23.845 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2021-12-08 15:16:23.849 UTC [90] LOG:  database system was shut down at 2021-12-08 15:16:23 UTC
2021-12-08 15:16:23.853 UTC [1] LOG:  database system is ready to accept connections
2021-12-08 15:16:23.895 UTC [99] FATAL:  password authentication failed for user "root"
2021-12-08 15:16:23.895 UTC [99] DETAIL:  Role "root" does not exist.
	Connection matched pg_hba.conf line 100: "host all all all scram-sha-256"
2021-12-08 15:16:32.646 UTC [112] FATAL:  database "main" does not exist
2021-12-08 15:16:32.663 UTC [113] FATAL:  database "main" does not exist
2021-12-08 15:16:32.682 UTC [114] FATAL:  database "main" does not exist
2021-12-08 15:16:32.701 UTC [115] FATAL:  database "main" does not exist
2021-12-08 15:16:32.718 UTC [116] FATAL:  database "main" does not exist
```
</details>

<details><summary> Click to Expand: PG Bouncer Container Log (running)</summary>

```logs

2021-12-08 15:16:24.501 UTC [1] LOG kernel file descriptor limit: 1048576 (hard: 1048576); max_client_conn: 100, max expected fd use: 232
2021-12-08 15:16:24.502 UTC [1] LOG listening on 0.0.0.0:5432
2021-12-08 15:16:24.502 UTC [1] LOG listening on unix:/tmp/.s.PGSQL.5432
2021-12-08 15:16:24.502 UTC [1] LOG process up: PgBouncer 1.15.0, libevent 2.1.12-stable (epoll), adns: udns 0.4, tls: OpenSSL 1.1.1l  24 Aug 2021
2021-12-08 15:16:25.241 UTC [1] LOG C-0x7ff4e594f110: postgres/supabase_auth_admin@172.26.0.5:50782 login attempt: db=postgres user=supabase_auth_admin tls=no
2021-12-08 15:16:25.242 UTC [1] LOG S-0x7ff4e5948130: postgres/supabase_auth_admin@172.26.0.2:5432 new connection to server (from 172.26.0.3:39860)
2021-12-08 15:16:25.244 UTC [1] ERROR S-0x7ff4e5948130: postgres/supabase_auth_admin@172.26.0.2:5432 cannot do SCRAM authentication: wrong password type
2021-12-08 15:16:25.244 UTC [1] LOG C-0x7ff4e594f110: postgres/supabase_auth_admin@172.26.0.5:50782 closing because: server login failed: wrong password type (age=0s)
2021-12-08 15:16:25.244 UTC [1] WARNING C-0x7ff4e594f110: postgres/supabase_auth_admin@172.26.0.5:50782 pooler error: server login failed: wrong password type
2021-12-08 15:16:25.244 UTC [1] LOG S-0x7ff4e5948130: postgres/supabase_auth_admin@172.26.0.2:5432 closing because: failed to answer authreq (age=0s)
2021-12-08 15:16:25.247 UTC [1] LOG C-0x7ff4e594f110: postgres/supabase_auth_admin@172.26.0.5:50786 login attempt: db=postgres user=supabase_auth_admin tls=no
2021-12-08 15:16:25.247 UTC [1] LOG C-0x7ff4e594f110: postgres/supabase_auth_admin@172.26.0.5:50786 closing because: pgbouncer cannot connect to server (age=0s)
2021-12-08 15:16:25.248 UTC [1] WARNING C-0x7ff4e594f110: postgres/supabase_auth_admin@172.26.0.5:50786 pooler error: pgbouncer cannot connect to server
2021-12-08 15:16:26.474 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42048 login attempt: db=postgres user=postgres tls=no
2021-12-08 15:16:26.475 UTC [1] LOG S-0x7ff4e5948130: postgres/postgres@172.26.0.2:5432 new connection to server (from 172.26.0.3:39866)
2021-12-08 15:16:26.479 UTC [1] ERROR S-0x7ff4e5948130: postgres/postgres@172.26.0.2:5432 cannot do SCRAM authentication: wrong password type
2021-12-08 15:16:26.479 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42048 closing because: server login failed: wrong password type (age=0s)
2021-12-08 15:16:26.479 UTC [1] WARNING C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42048 pooler error: server login failed: wrong password type
2021-12-08 15:16:26.479 UTC [1] LOG S-0x7ff4e5948130: postgres/postgres@172.26.0.2:5432 closing because: failed to answer authreq (age=0s)
2021-12-08 15:16:26.482 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42052 login attempt: db=postgres user=postgres tls=no
2021-12-08 15:16:26.482 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42052 closing because: pgbouncer cannot connect to server (age=0s)
2021-12-08 15:16:26.482 UTC [1] WARNING C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42052 pooler error: pgbouncer cannot connect to server
2021-12-08 15:16:27.443 UTC [1] LOG C-0x7ff4e594f110: postgres/supabase_storage_admin@172.26.0.8:41780 login attempt: db=postgres user=supabase_storage_admin tls=no
2021-12-08 15:16:27.445 UTC [1] LOG S-0x7ff4e5948130: postgres/supabase_storage_admin@172.26.0.2:5432 new connection to server (from 172.26.0.3:39872)
2021-12-08 15:16:27.447 UTC [1] ERROR S-0x7ff4e5948130: postgres/supabase_storage_admin@172.26.0.2:5432 cannot do SCRAM authentication: wrong password type
2021-12-08 15:16:27.447 UTC [1] LOG C-0x7ff4e594f110: postgres/supabase_storage_admin@172.26.0.8:41780 closing because: server login failed: wrong password type (age=0s)
2021-12-08 15:16:27.447 UTC [1] WARNING C-0x7ff4e594f110: postgres/supabase_storage_admin@172.26.0.8:41780 pooler error: server login failed: wrong password type
2021-12-08 15:16:27.447 UTC [1] LOG S-0x7ff4e5948130: postgres/supabase_storage_admin@172.26.0.2:5432 closing because: failed to answer authreq (age=0s)
2021-12-08 15:16:27.484 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42058 login attempt: db=postgres user=postgres tls=no
2021-12-08 15:16:27.484 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42058 closing because: pgbouncer cannot connect to server (age=0s)
2021-12-08 15:16:27.484 UTC [1] WARNING C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42058 pooler error: pgbouncer cannot connect to server
2021-12-08 15:16:29.486 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42062 login attempt: db=postgres user=postgres tls=no
2021-12-08 15:16:29.487 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42062 closing because: pgbouncer cannot connect to server (age=0s)
2021-12-08 15:16:29.487 UTC [1] WARNING C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42062 pooler error: pgbouncer cannot connect to server
2021-12-08 15:16:33.456 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42066 login attempt: db=postgres user=postgres tls=no
2021-12-08 15:16:33.456 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42066 closing because: pgbouncer cannot connect to server (age=0s)
2021-12-08 15:16:33.457 UTC [1] WARNING C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42066 pooler error: pgbouncer cannot connect to server
2021-12-08 15:16:41.460 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42072 login attempt: db=postgres user=postgres tls=no
2021-12-08 15:16:41.460 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42072 closing because: pgbouncer cannot connect to server (age=0s)
2021-12-08 15:16:41.460 UTC [1] WARNING C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42072 pooler error: pgbouncer cannot connect to server
2021-12-08 15:16:57.466 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42080 login attempt: db=postgres user=postgres tls=no
2021-12-08 15:16:57.466 UTC [1] LOG C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42080 closing because: pgbouncer cannot connect to server (age=0s)
2021-12-08 15:16:57.466 UTC [1] WARNING C-0x7ff4e594f110: postgres/postgres@172.26.0.7:42080 pooler error: pgbouncer cannot connect to server

```

</details>

<details><summary> Click to Expand: Auth Container Log (Exited 0)</summary>

```
[POP] 2021/12/08 15:16:25 info - 0.0113 seconds
time="2021-12-08T15:16:25Z" level=fatal msg="running db migrations: Migrator: problem creating schema migrations: couldn't start a new transaction: could not create new transaction: failed to connect to `host=supabase_pgbouncer_supabase_test user=supabase_auth_admin database=postgres`: server error (ERROR: pgbouncer cannot connect to server (SQLSTATE 08P01))"

```

</details>

<details><summary>Click to Expand: Storage Container Log (Exited 0)</summary>

```
running migrations
(node:8) UnhandledPromiseRejectionWarning: error: server login failed: wrong password type
    at Parser.parseErrorMessage (/snapshot/storage-api/node_modules/pg-protocol/dist/parser.js:278:15)
    at Parser.handlePacket (/snapshot/storage-api/node_modules/pg-protocol/dist/parser.js:126:29)
    at Parser.parse (/snapshot/storage-api/node_modules/pg-protocol/dist/parser.js:39:38)
    at Socket.<anonymous> (/snapshot/storage-api/node_modules/pg-protocol/dist/index.js:10:42)
    at Socket.emit (events.js:376:20)
    at addChunk (internal/streams/readable.js:309:12)
    at readableAddChunk (internal/streams/readable.js:284:9)
    at Socket.Readable.push (internal/streams/readable.js:223:10)
    at TCP.onStreamRead (internal/stream_base_commons.js:188:23)
(Use `storage-api-x86_64 --trace-warnings ...` to show where the warning was created)
(node:8) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:8) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

</details>

----
p.s. I'm a first time user so I apologize in advance if I'm missing an obvious step. Please let me know if I've to update some other configuration to make the supabase app work.

**Comment by soedirgo** [comment]
Thanks for the report! This should be fixed in v0.15.6 - can you see if that works on your end?

**Comment by osaxma** [comment]
Thank you! it's working now!
